### ISSUE TITLE
server+api interaction error: Body exeeded 1mb limit

### ISSUE THREAD

**Issue by FinnFrotscher** [inlife/nexrender#402]
**Describe the bug**
I am building a nexrender project and passing things like the template and some scripts  as base64 encoded strings. the resulting project is approximately 9mb in size. using the @nexrender/api i intended to send this object to an instance of @nexrender/server running inside a docker image. 

the server responded with:
```
[Symbol(Response internals)]: {
    url: 'https://nexrender.com/api/v1/jobs',
    status: 413,
    statusText: 'Request Entity Too Large',
    headers: Headers { [Symbol(map)]: [Object: null prototype] },
    counter: 0
  }
}
UnhandledPromiseRejectionWarning: Error: Body exceeded 1mb limit
```

**Comment by FinnFrotscher** [comment]
the problem is that the server uses micro under the hood, which in (which sets a default of 1 mb)[https://github.com/vercel/micro/blob/98611027286831742ac407e15d19ad35ca8baf11/packages/micro/lib/index.js#L152-L159]

**Comment by FinnFrotscher** [comment]
thus nexrender-server needs to either provide a more generous default [here](https://github.com/inlife/nexrender/blob/6ea877aef6c5e6e73ebbe6ca8a79511e234149d3/packages/nexrender-server/src/routes/jobs-create.js#L7) or we need to make this a cli param

**Comment by FinnFrotscher** [comment]
set default value.
create pull request. 
https://github.com/inlife/nexrender/issues/402

**Comment by inlife** [comment]
Very good point @FinnFrotscher, and thank you for the fix! :)

**Comment by inlife** [comment]
Published as `1.16.4`

**Comment by FinnFrotscher** [comment]
i tried to  set the limit as a cli argument, but there is no way to get the the variable down to the route function. maybe some  centralized storage for things like limit or the  secrets could come in handy. just an  idea.
